packages/cactus-plugin-ledger-connector-corda/package.json

```diff
@@ -13,7 +13,8 @@
   ],
   "scripts": {
     "generate-sdk": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g typescript-axios -o ./src/main/typescript/generated/openapi/typescript-axios/",
-    "generate-server": "openapi-generator-cli generate -i ./src/main/json/openapi.json -g kotlin-spring -o ./src/main-server/kotlin/gen/kotlin-spring/ -c ./src/main-server/openapi-generator-config.yaml",
+    "clean-server": "del-cli 'src/main-server/kotlin/gen/kotlin-spring/src/**/{model,api}/*'",
```

Comment:
@elenaizaguirre Sorry for the nit pick as usual. You could add this deletion pattern to the root `clean` script and then we don't need to deviate as much from how the build system works in general:
`'src/main-server/kotlin/gen/kotlin-spring/src/**/{model,api}/*'`

(there's already a list of patterns in that clean script and I'm concerned that it's getting longer and longer as well, but at least if we have the pattern there then all the patterns are at the same place for newcomers it should be easier to discover/understand what gets deleted and when during the build process)